tests/customer-resource-custom-coverage-test.js

```diff
@@ -116,11 +116,10 @@ describeApplication('CustomerResourceCustomCoverage', () => {
 
         describe('clicking the clear row button', () => {
           beforeEach(() => {
-            return convergeOn(() => {
-              expect(CustomerResourceCoverage.dateRangeRowList.length).to.equal(2);
-            }).then(() => {
-              CustomerResourceCoverage.dateRangeRowList[1].clickRemoveRowButton();
-            });
+            return new Convergence()
+              .once(() => expect(CustomerResourceCoverage.dateRangeRowList.length).to.equal(2))
```

Comment:
Stylistically speaking, we don't need to use an assertion here. and just use the boolean condition. Of course, the expectation _should_ give better error messages.